src/components/button/button.component.ts

```diff
@@ -46,7 +46,7 @@ export class OsButton implements IOsButton {
   }
 
   setDisabled() {
-    if (this.disabled) {
+    if (this.disabled || this.loading) {
```

Comment:
No it doesn't, but when checking this I have just noticed that I could add a couple tests for this new logic, going to add another commit with this in.